### PR TITLE
added timeline translation for hw

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

File: ui/raidboss/data/03-hw/raid/a12s.js [hunk 1]
```diff
@@ -5,4 +5,246 @@
...
+        'Wings Spawn': 'Wings Spawn', // FIXME
+      },
+    },
+    {
+      'locale': 'ja',
+      'replaceSync': {
+        'Removing combatant Alexander Prime': 'Removing combatant Alexander Prime', // FIXME
+        'casts Holy Bleed': 'casts Holy Bleed', // FIXME
+        'casts Void of Repentance': 'casts Void of Repentance', // FIXME
+        'readies Chronofoil': 'readies Chronofoil', // FIXME
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/raid/a12s.js]: Oof.  At some point before the next expansion, I was going to add better alex normal and savage timelines and triggers.  These are all really old and not made by me.  I think I would almost rather just leave all of Alex savage alone for now and fix them up once they're better.

Comment (Akurosia) [reply ui/raidboss/data/03-hw/raid/a12s.js]: So are you fine if i just remove all alex from this PR?

Comment (quisquous) [reply ui/raidboss/data/03-hw/raid/a12s.js]: Yeah, I think removing all of alex is the best at this point.  Sorry for the churn.  I should have said something earlier.

Comment (Akurosia) [reply ui/raidboss/data/03-hw/raid/a12s.js]: ok i removed them for now. Will open an issue so we can keep track.

File: ui/raidboss/data/03-hw/trial/sephirot-ex.js [hunk 2]
```diff
@@ -5,4 +5,211 @@
...
+        'Yesod, MOVE': 'Yesod, MOVE', // FIXME
+      },
+    },
+    {
+      'locale': 'ja',
+      'replaceSync': {
+        'Added new combatant Cochma': 'Added new combatant Cochma', // FIXME
+        'Ein Sof Ohr': 'アイン・ソフ・オウル',
+        'Removing combatant Sephirot': 'Removing combatant Sephirot', // FIXME
+        'uses Triple Trial': 'uses Triple Trial', // FIXME
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/trial/sephirot-ex.js]: This too.

File: ui/raidboss/data/03-hw/trial/ravana-ex.js [hunk 3]
```diff
@@ -5,4 +5,241 @@
...
   triggers: [
   ],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Dance to the song of ringing steel': 'Dance to the song of ringing steel', // FIXME
+        'Ravana readies The Seeing': 'Ravana readies The Seeing', // FIXME
+        'Ravana starts using Bloody Fuller on Ravana': 'Ravana starts using Bloody Fuller on Ravana', // FIXME
+        'Ravana starts using Final Liberation on Ravana': 'Ravana starts using Final Liberation on Ravana', // FIXME
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/trial/ravana-ex.js]: This is the same as the Alex stuff.  I would really rather not have replacement syncs that have full lines in them.

**Comment by quisquous** [comment]
I think this is also all very hard to review with all of the reordering in terms of whether the ordering is important or not.  I think I would like to write some timeline tests that can maybe catch any errors this might create before merging this and the sb patches.

File: ui/raidboss/data/03-hw/trial/ravana-ex.js (hunk 3 above)
Comment (quisquous) [reply ui/raidboss/data/03-hw/trial/ravana-ex.js]: This looks like it's still an issue.

File: ui/raidboss/data/03-hw/trial/sephirot-ex.js [hunk 4]
```diff
@@ -5,4 +5,211 @@
   timelineFile: 'sephirot-ex.txt',
   triggers: [
   ],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Added new combatant Cochma': 'Added new combatant Cochma', // FIXME
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/trial/sephirot-ex.js]: This should just be "Cochma" and "Sephirot" below.

File: ui/raidboss/data/03-hw/trial/sephirot-ex.js [hunk 5]
```diff
@@ -5,4 +5,211 @@
...
+        'Ein Sof Ohr': 'En Sof Or',
+        'Removing combatant Sephirot': 'Removing combatant Sephirot', // FIXME
+        'uses Triple Trial': 'uses Triple Trial', // FIXME
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/trial/sephirot-ex.js]: This is another "using" that should be fixed in the timeline.

File: ui/raidboss/data/03-hw/trial/sophia-ex.js [hunk 6]
```diff
@@ -5,4 +5,191 @@
   timelineFile: 'sophia-ex.txt',
   triggers: [
   ],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Added new combatant Aion Teleos': 'Added new combatant Aion Teleos', // FIXME
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/trial/sophia-ex.js]: All the same stuff here.

**Pull request merged**